### DISCUSSION TITLE
Control and Data-Plane Configuration

### DISCUSSION THREAD

**Discussion by HendrikHahn1996** [eclipse-tractusx/tractusx-edc#1472] [Q&A]
Hello!

For setting up the control plane and the data plane it requires a list of configurations (configuration.properties).

The is list is quite long and I am not sure where to get all these information from. 

Is there any documentation about the configurations? 

Thanks!

**Comment by wolf4ood** [comment]
Hi @HendrikHahn1996 

we have the documentation for configuring CP and DP using our official helm charts

For example this one for Postgres + Hashicorp vault distribution

https://github.com/eclipse-tractusx/tractusx-edc/blob/main/charts/tractusx-connector/README.md

Thanks

**Comment by nikp86** [comment]
Hi @HendrikHahn1996 ,
the easiest way to start the connector is to use the helm charts and provide the following information in the values file:

- iatp.sts.oauth.token_url: the token endpoint of DIM
- iatp.sts.oauth.client.id: the client ID of your tenant in DIM
- iatp.sts.oauth.client.secret_alias: alias under which you saved your DIM client secret in the vault
- iatp.sts.dim.url: the base URL for DIM
- controlplane.bdrs.server.url: base URL of the BPN-DID Resolution Service ("BDRS")
- iatp.id
- participant.id

Also please store client secret in the HashiCorp vault using an alias. But by default, Tractus-X EDC expects to find the secret under secret/client-secret. The alias must be configured using the iatp.sts.oauth.client.secret_alias Helm value.

As reference please check below sample file: https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/deployment/src/main/resources/helm/tractusx-connector-test.yaml

Hope the information helps :)
Best Regards 
Nik

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
